poster/styles/styles.scss

```diff
@@ -146,26 +217,55 @@ ul{
   grid-column: 6/8;
   grid-row: 1/3;
   text-align: center;
+
+  @media only screen and (max-width: 700px){
+    grid-row: 1/2;
+  }
+
+  @media only screen and (max-width: 700px){
+    grid-column: 2/9;
+    padding-right: $spacing-margin;
+  }
 }
 
 .title__place{
-  grid-column: 2/5;
+  grid-column: 1/5;
   grid-row: 3/4;
   text-align: center;
-}
 
-.caption__madegoods{
-  grid-column: 2/4;
-  grid-row: 1/2;
-  text-align: center;
+  @media only screen and (max-width: 730px) {
+    padding-left: $spacing-margin;
+  }
+
+  @media only screen and (max-width: 700px){
+    grid-column: 1/7;
+  }
+
+  @media only screen and (max-width: 500px){
+    grid-column: 1/8;
+  }
+
 }
 
+// lists
+
 .list__1{
```

Comment:
All the list classes have some characteristics in common and those would be helpful to combine in another class.